helix-core/src/test/java/org/apache/helix/manager/zk/TestZkBucketDataAccessor.java

```diff
@@ -0,0 +1,126 @@
+package org.apache.helix.manager.zk;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import org.apache.helix.BucketDataAccessor;
+import org.apache.helix.HelixProperty;
+import org.apache.helix.TestHelper;
+import org.apache.helix.ZNRecord;
+import org.apache.helix.common.ZkTestBase;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class TestZkBucketDataAccessor extends ZkTestBase {
+
+  private String PATH = "/" + TestHelper.getTestClassName();
+  private String NAME_KEY = TestHelper.getTestClassName();
+  private List<String> LIST_FIELD;
+  private Map<String, String> MAP_FIELD;
+
+  private BucketDataAccessor _bucketDataAccessor;
+
+  @BeforeClass
+  public void beforeClass() {
+    _bucketDataAccessor = new ZkBucketDataAccessor(ZK_ADDR);
+
+    // Populate LIST_FIELD for R/W comparison
+    LIST_FIELD = new ArrayList<>();
```

Comment:
Can't you get rid of the beforeClass method and just set it inline?

For example,

```
private static final List<String> LIST_FIELDS = ImmutableList.of("1", "2");
private static final Map<String, String> MAP_FIELD = ImmutableMap.of("1", "2");
```